osu.Game/Online/Spectator/SpectatorClient.cs

```diff
@@ -33,7 +33,7 @@ public abstract partial class SpectatorClient : Component, ISpectatorClient
         /// Whether the <see cref="SpectatorClient"/> is currently connected.
         /// This is NOT thread safe and usage should be scheduled.
         /// </summary>
-        public abstract IBindable<bool> IsConnected { get; }
+        protected abstract IBindable<bool> IsConnected { get; }
```

Comment:
Any particular reason for encapsulating this? I rely on this being public in https://github.com/ppy/osu/pull/25522.

Reply:
No reason, just wasn't being used.

Reply:
I guess I can just re-expose it again in mine when resolving merge conflicts.